src/main/java/controllers/CommandLineInterface.java

```diff
@@ -322,7 +325,7 @@ public Schedule displayUserSchedules(List<Schedule> userSchedules) {
                     break;
                 case 'S':
                     System.out.println("Saving this schedule in .ics format...");
-                    ScheduleExporter.outputScheduleICS(currSchedule);
+                    new ICSExporter().outputSchedule(currSchedule);
```

Comment:
im guessing the file name option will be added in the refactoring

Reply:
yeah, i operated under that assumption. I'm gonna add that as a note to the Ghub projects card.